packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -57,12 +57,8 @@ const gcOptionsMatrix: OptionsMatrix<IGCRuntimeOptions> = {
 };
 
 const summaryOptionsMatrix: OptionsMatrix<ISummaryRuntimeOptions> = {
-    disableSummaries: [false],
     initialSummarizerDelayMs: numberCases,
     summaryConfigOverrides: [undefined],
```

Comment:
You need to fill in these values otherwise summaries will always run with the default values in stress tests. We want the containers in the test to run with different (random) set of configurations.

Reply:
I tried to address it on the next iteration - please let me know what you think - We will gradually need to add some sets to each property as I don't suppose we want random numbers in all of them.